python/dgl/transforms/module.py

```diff
@@ -98,6 +100,162 @@ def __call__(self, g):
     def __repr__(self):
         return self.__class__.__name__ + '()'
 
+class NormalizeFeatures(BaseTransform):
+    r"""
+    Row-normalizes the features given in `node_feat_names` and `edge_feat_names` to sum-up to one.
+
+    Parameters
+    ----------
+    node_feat_names : list of str, optional
+        The names of the node features to be normalized. If None, all node features will be normalized.
+        Default: `None`.
+    edge_feat_names : list of str, optional
+        The names of the edge features to be normalized. If None, all edge features will be normalized.
+        Default: `None`.
+
+    Example
+    -------
+
+    >>> import dgl
+    >>> import torch
+    >>> from dgl import NormalizeFeatures
+
+    Case1: Normalize features of a homogenous graph.
```

Comment:
It seems that people use `homogeneous` more often in the GNN field.

Reply:
Got it, I fixed it.